config/log.conf

```diff
@@ -1,6 +1,27 @@
-log_format main '$remote_addr - $remote_user [$time_local]  $status '
-                '"$request" $body_bytes_sent "$http_referer" '
-                '"$http_user_agent" "$http_x_forwarded_for" $proxy_x_request_id';
+log_format main_default escape=default
+  '$remote_addr - $remote_user [$time_local] $status '
+  '"$request" $body_bytes_sent "$http_referer" '
+  '"$http_user_agent" "$http_x_forwarded_for" $proxy_x_request_id';
 
-access_log /dev/stdout main;
+log_format main_json escape=json
+  '{'
+    '"body_bytes_sent":"$body_bytes_sent",'
+    '"host":"$host",'
+    '"http_referrer":"$http_referer",'
+    '"http_user_agent":"$http_user_agent",'
+    '"http_x_forwarded_for":"$http_x_forwarded_for",'
+    '"proxy_x_forwarded_port":"$proxy_x_forwarded_port",'
+    '"proxy_x_forwarded_proto":"$proxy_x_forwarded_proto",'
+    '"proxy_x_forwarded_ssl":"$proxy_x_forwarded_ssl",'
+    '"proxy_x_request_id":"$proxy_x_request_id",'
+    '"remote_addr":"$remote_addr",'
+    '"remote_user":"$remote_user",'
+    '"request":"$request",'
+    '"request_length":"$request_length",'
+    '"request_time":"$request_time",'
+    '"status": "$status",'
+    '"time_local":"$time_local"'
```

Comment:
By the way this returns `10/Feb/2021:12:31:06 +000` but there's also `$time_iso8601`. Thoughts?

```
$time_iso8601
    local time in the ISO 8601 standard format
$time_local
    local time in the Common Log Format
```

Reply:
Options are:

1. We change to `$time_iso8601`.
2. We keep both `$time_iso8601` and `$time_local`.

I'm leaning on the second one so that one can choose what they want.

Reply:
I think `$time_iso8601` is better as more common than CLF. 😬

Reply:
LOL. Ok, changing.

Reply:
Changed to `$time_iso8601` in https://github.com/Intellection/docker-nginx-proxy/commit/c2b081f4bd75997491db3e73b8892999ba4f9707.